controllers/grid/navigationMenus/form/NavigationMenuItemsForm.php

```diff
@@ -45,9 +45,11 @@ public function fetch($request, $template = null, $display = false)
             ->getCollector()
             ->filterByContextIds([$contextId])
             ->getMany();
-        $seriesTitles = $series->map(fn (Section $seriesObj) => [
-            $seriesObj->getId() => $seriesObj->getLocalizedTitle()
-        ]);
+
+        $seriesTitles = [];
+        foreach ($series as $seriesObj) {
+            $seriesTitles[$seriesObj->getId()] = $seriesObj->getLocalizedTitle();
+        }
```

Comment:
I think the previous code can be kept, but fixed.

```suggestion
        $seriesTitles = $series->mapWithKeys(fn (Section $seriesObj) => [
            $seriesObj->getId() => $seriesObj->getLocalizedTitle()
        ])->all();
```